modules/apps/portal-cache/portal-cache-impl/src/main/java/com/liferay/portal/cache/internal/dao/orm/FinderCacheImpl.java

```diff
@@ -502,6 +491,22 @@ protected void activate(BundleContext bundleContext) {
 
 		_serviceTrackerMap = ServiceTrackerMapFactory.openSingleValueMap(
 			bundleContext, ArgumentsResolver.class, "class.name");
+
+		_serviceRegistration = bundleContext.registerService(
+			CacheRegistryItem.class, new FinderCacheCacheRegistryItem(), null);
+	}
+
+	@Deactivate
+	protected void deactivate() {
+		_serviceRegistration.unregister();
+	}
+
+	private void _cleanCache() {
+		clearLocalCache();
```

Comment:
Hey @GabrielHenriquedev, you do not need to create this private method. You can call the `clearCache` directly into the `FinderCacheCacheRegistryItem` class.